Library/Homebrew/keg_relocate.rb

```diff
@@ -16,9 +16,10 @@ def fix_dynamic_linkage
       link = file.readlink
       # Don't fix relative symlinks
       next unless link.absolute?
-      if link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
-        FileUtils.ln_sf(link.relative_path_from(file.parent), file)
-      end
+      next unless link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
```

Comment:
I don't think I can; `next if a || b` can be written `next if a; next if b` and `next unless a && b` can be written `next unless a; next unless b` but I don't see what to do here. My boolean logic or my imagination might be failing me.